src/inject/fallback.ts

```diff
@@ -1,11 +1,31 @@
+import {createNodeAsap} from './utils/dom';
+
 if (
```

Comment:
Let's add `document.documentElement instanceof HTMLHtmlElement` condition, so that it will not inject anything into SVGs and other strange files.